### ISSUE TITLE
Poke-Spawn point dissapeared going from vers. 1.0.0 to 2.1.0

### ISSUE THREAD

**Issue by Cawleery** [AHAAAAAAA/PokemonGo-Map#3223]
Hello, Ive got 2 spawn points within a few hundred meters from my house (live in the middle of nowhere) and the closest one of them dissapeared when i went from version 1.0.0 to 2.1.0.
Vers. 1.0.0 still finds the pokemons there but when i use the 2.1.0 version it wont find the pokemon eventhough i know it is there (checking both 1.0.0 and IRL). 
Ive switched the green circles on in 2.1.0 so i know it overlaps the area and ive tried both larger and smaller ammounts of steps (4-7).

The aprox coords for this spawnpoint is: 58.5026,12.2285

This is the only spot i found so far not existing in both versions.

Using win 7 for both and the other programmes req and listed in the wiki

Thanks for help in advance

**Comment by hack3rjack** [comment]
Also finding that some pokemon are not being found when scanned in master and development (single and beehive), when a pass has been made several times over the spot I just caught them on app...not a single occurrence.

**Comment by Cawleery** [comment]
Hmm even more wierd, managed to do some more "reserch" on my own and found out that if i put the coords on exactly 58.5026,12.2285 it will find the pokemons on the spawn point. 
But not if i put the coords a bit further away.
Tried swapping between 58.5026,12.2285 (the spawn point) and 58.5049,12.2240 (closer to my house) and it does work on the first but not the second.

**Comment by kneel23** [comment]
right, ever since the update the Pokemon showing in the scanner are few and far between.   I have a theory, that is that Niantic basically created a second layer of "poke-spawners" that the newest app version knows how to decipher, but we have yet to reverse-engineer (not speaking on behalf of the developers here, just me guessing out my ass).  Question remains why would they even bother leaving the version 1 pokemon spawning?  My guess at an answer to that is for people who haven't or won't update, that way they still see SOME pokemon.   In time this will likely change again

**Comment by Cawleery** [comment]
I see that in cmd it says:

2016-08-03 13:13:55,315 [        search] [   INFO] Search loop 14 starting
2016-08-03 13:13:55,315 [        search] [   INFO] Skipping Pokemon Go login process since already logged in                 for another 1257.68 seconds
2016-08-03 13:13:55,530 [        models] [   INFO] Upserted **1** pokemon, 0 pokestops, and 0 gyms
2016-08-03 13:14:00,739 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 13:14:05,951 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 13:14:11,164 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 13:14:16,397 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 13:14:21,664 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms
2016-08-03 13:14:26,891 [        models] [   INFO] Upserted 0 pokemon, 0 pokestops, and 0 gyms


But this pokemon wont show on map. But i went out to see if it was there and it was.

**Comment by hack3rjack** [comment]
@Cawleery which pokemon do you have filtered out? Which pokemon was there on app but not on scanner? 

@AHAAAAAAA is there any conversation on this already where pokemon are being scanned but not added to map? Could there be a bug introduced when pokemon are hidden/filtered? Or maybe there are pokemon written to db but not to map, or gaps missed when multiple workers/threads are running? I noticed missing pokemon around the time the dev build for multiple workers/behive was added, for example I saw a Ninetales yesterday in the app pop up next to me, so I set location by dragging the map marker to my current location but it never found him (no lures/incense, etc).

**Comment by Cawleery** [comment]
@hack3rjack I have no pokemons hidden at all. The pokemons ive found out there IRL is Venomoth, Rattata and Weedle i think, maybe even a sparrow. Cause those are the ones that usually spawn there. Now last time i checked it was a Venomoth not showing at least, that i am sure of.
